docs/web3-eth.rst

```diff
@@ -144,7 +144,7 @@ Property
 --------
 
 
-``String`` - 20 Bytes: Any ethereum address. You should have the private key for that address in your node or keystore. (Default is ``undefined``)
+``String`` - 20 Bytes: Any Ethereum address. You should have the private key for that address in your node or keystore. (Default is ``undefined``)
```

Comment:
Use "need":

``String`` - 20 Bytes: Any Ethereum address. You need to have the private key for that address in your node or keystore. (Default is ``undefined``)